src/Microsoft.ServiceFabric.Actors.KVSToRCMigration/ActorStateMigratedDataValidationFailedException.cs

```diff
@@ -0,0 +1,35 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Actors.KVSToRCMigration
+{
+    using System;
+    using System.Fabric;
+    using System.Runtime.Serialization;
+
+    [Serializable]
+    internal sealed class ActorStateMigratedDataValidationFailedException : FabricException
```

Comment:
Since this is an exception class, there is no necessity of calling out failure explicitly in the name. The name of the class could be MigrationDataValidationException. Thoughts?

Reply:
makes sense